manifests/m/Microsoft/XNARedist/4.0.20823/Microsoft.XNARedist.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.XNARedist
+PackageVersion: 4.0.20823

```

Comment:
```suggestion
PackageVersion: 4.0.20823.0
```

Reply:
`PackageVersion` must match Control Panel. You'll also have to update the folder from `4.0.20823` to `4.0.20823.0`